content/blog/2021-05-04-prometheus-conformance-remote-write-compliance.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Prometheus conformance program: Remote Write Compliance test results"
+created_at: 2021-05-05
+kind: article
+author_name: Richard "RichiH" Hartmann
+---
+
+As [announced by CNCF](https://www.cncf.io/blog/2021/05/03/announcing-the-intent-to-form-the-prometheus-conformance-program/) and by [ourselves](https://prometheus.io/blog/2021/05/03/introducing-prometheus-conformance-program/), we're starting a Prometheus conformance program.
+
+To give everyone an overview of where the ecosystem is before running tests officially, we wanted to show off the newest addition to our happy little bunch of test suites: The Prometheus Remote Write tests.
+
+During Monday's [PromCon](https://promcon.io/2021-online/), Tom presented the test results from the time of recording a few weeks ago. In the live section, he already had an [update](https://docs.google.com/presentation/d/1RcN58LlS3V5tYCUsftqUvNuCpCsgGR2P7-GoH1MVL0Q/edit#slide=id.gd1789c7f7c_0_0). Two days later we have two more updates: The addition of [vector](https://github.com/prometheus/compliance/pull/24) and [new versions](https://github.com/prometheus/compliance/pull/25).
```

Comment:
"vector" is called "Vector" with a capital V.

I'd not require people to click on the link to find out what Vector is. How about:

"The addition of the observability pipeline tool Vector, as well as new versions of existing systems."

Reply:
done